src/Sylius/Behat/Page/Admin/ProductOption/CreatePage.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Page\Admin\ProductOption;
+
+use Behat\Mink\Element\Element;
+use Behat\Mink\Exception\ElementNotFoundException;
+use Sylius\Behat\Behaviour\SpecifiesItsCode;
+use Sylius\Behat\Page\Admin\Crud\CreatePage as BaseCreatePage;
+
+/**
+ * @author Grzegorz Sadowski <grzegorz.sadowski@lakion.com>
+ */
+class CreatePage extends BaseCreatePage implements CreatePageInterface
+{
+    use SpecifiesItsCode;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function nameItIn($name, $language)
+    {
+        $this->getDocument()->fillField(
+            sprintf('sylius_product_option_translations_%s_name', $language), $name
+        );
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function addOptionValue($code, $value)
+    {
+        $this->getDocument()->clickLink('Add value');
+
+        $optionValueForm = $this->getLastOptionValueElement();
+
+        $optionValueForm->fillField('Code', $code);
+        $optionValueForm->fillField('Value', $value);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function checkValidationMessageForOptionValues($message)
+    {
+        $optionValuesValidationElement = $this->getElement('ui_segment')->find('css', '.ui.pointing');
+        if (null === $optionValuesValidationElement) {
+            throw new ElementNotFoundException($this->getDriver(), 'product option validation box', 'css', '.ui.pointing');
+        }
+
+        return $optionValuesValidationElement->getText() === $message;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getDefinedElements()
+    {
+        return array_merge(parent::getDefinedElements(), [
+            'code' => '#sylius_product_option_code',
+            'name' => '#sylius_product_option_translations_en_US_name',
+            'values' => '#sylius_product_option_values',
+            'ui_segment' => '.ui.segment',
```

Comment:
``ui_segment`` is not the best name for page element :/ I have no idea what it is (previous elements have meaningful names).